src/sugar/autofill.ts

```diff
@@ -132,6 +130,17 @@ async function setNextValidSequenceNumber(
   tx.Sequence = data.result.account_data.Sequence
 }
 
+async function fetchAccountDeleteFee(client: Client): Promise<BigNumber> {
+  const response = await client.request({ command: 'server_state' })
```

Comment:
What's the difference between `server_state` and `server_info`?

Reply:
Per [the documentation](https://xrpl.org/server_state.html) (emphasis added):

> The server_state command asks the server for various machine-readable information about the rippled server's current state. The response is almost the same as the server_info method, but uses units that are easier to process instead of easier to read. (For example, **XRP values are given in integer drops** instead of scientific notation or decimal values, and time is given in milliseconds instead of seconds.)